Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -278,17 +284,30 @@ extension CrashLogging {
     }
 }
 
-// MARK: - Event Logging
-extension Event {
+internal extension TracksUser {
```

Comment:
This is the code that used to be in TracksUser.swift. It's just moved directly, with no changes.

Reply:
Yeah, `internal` is the default. I sometimes do this just to make it clear to the reader that these methods are not being exposed publicly.